src/block-management/inline-toolbar/index.js

```diff
@@ -52,19 +54,42 @@ export default class InlineToolbar extends React.Component<PropsType> {
 	}
 
 	render() {
+		const { order } = this.props;
+		const moveUpButtonTitle = this.props.canMoveUp ? sprintf(
+			/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+			__( 'Move up from row %1$s to row %2$s' ),
+			order,
+			order - 1 ) : sprintf(
+			/* translators: accessibility text. %d: block position (number) */
+			__( 'Move up from row %1$s' ),
+			order );
+
+		const moveDownButtonTitle = this.props.canMoveUp ? sprintf(
+			/* translators: accessibility text. %1: current block position (number). %2: next block position (number) */
+			__( 'Move down from row %1$s to row %2$s' ),
+			order,
+			order + 1 ) : sprintf(
+			/* translators: accessibility text. %d: block position (number) */
+			__( 'Move down from row %1$s' ),
+			order );
+
+		const removeButtonTitle = sprintf(
+			/* translators: accessibility text. %1: current block position (number). */
+			__( 'Remove row %1$s' ),
```

Comment:
I think we had previously discussed taking the word block out of the text since the block name would have already been said in VoiceOver once the block is selected, WDYT?